libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKOAuth2.h

```diff
@@ -87,7 +87,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) BOOL hasError;
 @property (nonatomic, readonly, nullable) SFSDKOAuthTokenEndpointErrorResponse *error;
 @property (nonatomic, readonly) NSString *accessToken;
-@property (nonatomic, readonly) NSString *refreshToken;
+@property (nonatomic, readonly, nullable) NSString *refreshToken;
```

Comment:
Refresh token should be `nonnull`.

Reply:
@bhariharan Im reusing the token endpoint call.  The refreshToken is not  returned during the refresh flow so it has to be optional right?

Reply:
Right - but it would still have the old value, right? If an app retrieves this `property`, it should always have a value, right?

Reply:
@bhariharan Sounds good to me.